src/main/java/com/github/maven_nar/cpptasks/compiler/CommandLineCompiler.java

```diff
@@ -134,17 +140,24 @@ protected void buildDefineArguments(CompilerDef[] defs, Vector<String> args) {
     }
     /**
      * Compiles a source file.
+     * @param commandPrefixes 
      * 
      */
     public void compile(CCTask task, File outputDir, String[] sourceFiles,
             String[] args, String[] endArgs, boolean relentless,
-            CommandLineCompilerConfiguration config, ProgressMonitor monitor)
+            CommandLineCompilerConfiguration config, ProgressMonitor monitor, boolean ccache)
```

Comment:
No problem! What do you mean by redirects?

I assumed these were internal methods but I guess their public nature means you could have a dependent project?